charts/kubewarden-controller/chart-values.yaml

```diff
@@ -110,3 +110,4 @@ experimental:
       successfulJobsHistoryLimit: 3
     containerRestartPolicy: Never
     skipAdditionalNamespaces: []
+    logLevel: "info"
```

Comment:
I remember Flavio preferring to hardcode this, and I share the same opinion. I don't think it provides anything useful for end users, they shouldn't have the need to change it.